onnxruntime/core/framework/op_node_proto_helper.cc

```diff
@@ -37,6 +37,32 @@ inline bool HasTyped<GraphProto>(const AttributeProto* attr) {
   return utils::HasGraph(*attr);
 }
 
+template <typename T>
+inline bool HasTypedSpan(const AttributeProto* attr);
```

Comment:
nit: gsl:span is how were returning it. Do you mean to say HasTypedList here?

Reply:
Good suggestion.

---
In reply to: [513918740](https://github.com/microsoft/onnxruntime/pull/5624#discussion_r513918740) [](ancestors = 513918740)